### ISSUE TITLE
some questions about the network structure?

### ISSUE THREAD

**Issue by wqz960** [huawei-noah/ghostnet#25]
Hi! Thank you for your codes!
There are some differences in your code. In your paper, after AvgPooling, the features maps go through a conv1x1 and a fc layer to output the result, but in your pytorch version, the feature maps go through two fc layers, and in your tensorflow version, the feature maps seems to go through two conv1x1(fully convolution network), which one can produce the highest accuracy and reach the accuracy written in your paper?
Thank you!

**Comment by iamhankai** [comment]
The conv1x1 is equivalent to FC layer if the input is of spatial size (1x1).
